### DISCUSSION TITLE
Suggested memcached server settings

### DISCUSSION THREAD

**Discussion by ngdaddikar** [express-rate-limit/rate-limit-memcached#11] [Q&A]
Are there any recommended Memcached server settings e.g. max-item-size.

Thanks
Nikhil

**Comment by nfriedly** [comment]
I think the defaults work well for most users. This store just saves two numbers per user (hit count and expiration time), so `max-item-size` shouldn't have much effect.

I'd like to find a memcached client that supports the newer [meta commands](https://github.com/memcached/memcached/wiki/MetaCommands), then it can be reduced to a single item (the expiration time can be read from the metadata).
